3rd-task/index.html

```diff
@@ -0,0 +1,374 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="style.css" rel="stylesheet">
+    <title>3rd Task</title>
+
+</head>
+<body>
+<div id="wrapper">
+    <header class="nav-header flex">
+    <nav id="nav" class="container flex">
+    <ul class="flex">
+        <li><a href="index.html">Home</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Features</a></li>
+        <li><a href="#">Services</a></li>
```

Comment:
Adding `#` in `href` will scroll our page to the top. Can we please fix this one ?

Reply:
we have design just main home page design